src/component/keyboard/ShowPressedButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+type State = {
+  key: string | null;
+};
+
+export class ShowPressedButton extends React.Component {
+  state: Readonly<State> = {
+    key: null,
```

Comment:
we can start with empty string, null & undefined not user-friendly with UI/UX
```suggestion
    key: '',
```